SweetTweetApp/src/main/java/JSONDecorator/JSONDecorator.java

```diff
@@ -1,14 +1,14 @@
-package servletsForPosts;
+package JSONDecorator;
```

Comment:
There's a convention, that packages in Java named in small latters only. Your IntelliJ IDEA will even complain for that